packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -146,6 +156,9 @@ BpkCalendarNav.propTypes = {
   maxDate: PropTypes.instanceOf(Date).isRequired,
   minDate: PropTypes.instanceOf(Date).isRequired,
   month: PropTypes.instanceOf(Date).isRequired,
+  nextMonthLabel: PropTypes.string.isRequired,
+  nowDisplayingLabel: PropTypes.string.isRequired,
+  previousMonthLabel: PropTypes.string.isRequired,
```

Comment:
This is 4 labels total now that are being passed down to the NAV from the datepicker, how do others feel about that?